### PR TITLE
Big_tiff

### PR DESCRIPTION
**Pull request by jdroenner**

### CODE REVIEW

**Comment by jdroenner** [comment]
[Immer BigTiff verwenden](https://trello.com/c/pPfS5dXI/603-immer-bigtiff-verwenden)

File: operators/src/util/raster_stream_to_geotiff.rs [hunk 1]
```diff
@@ -113,6 +113,7 @@ where
 const COG_BLOCK_SIZE: &str = "512";
 const COMPRESSION_FORMAT: &str = "LZW";
 const COMPRESSION_LEVEL: &str = "9"; // maximum compression
+const BIG_TIFF_BYTES: usize = 2_000_000_000; // ~ 2GB + 2GB for overviews + buffer for headers
```
Comment (ChristianBeilschmidt) [inline operators/src/util/raster_stream_to_geotiff.rs:116]:
```suggestion
const BIG_TIFF_BYTE_THRESHOLD: usize = 2_000_000_000; // ~ 2GB + 2GB for overviews + buffer for headers
```

File: operators/src/util/raster_stream_to_geotiff.rs [hunk 2]
```diff
@@ -154,8 +155,15 @@ impl<P: Pixel + GdalType> GdalDatasetWriter<P> {
         );
         let output_bounds = query_rect.spatial_bounds;
 
+        let byte_size = width as usize * height as usize * std::mem::size_of::<P>();
```
Comment (ChristianBeilschmidt) [inline operators/src/util/raster_stream_to_geotiff.rs:158]:
```suggestion
        let uncompressed_byte_size = width as usize * height as usize * std::mem::size_of::<P>();
```

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]
Maybe indicate the threshold and heuristic more with variable names. Nothing else to complain. Should try it out after it is merged.

File: operators/src/util/raster_stream_to_geotiff.rs (hunk 2 above)
Comment (jdroenner) [reply operators/src/util/raster_stream_to_geotiff.rs]: [55358bb](https://github.com/geo-engine/geoengine/pull/473/commits/55358bb33360ff88ff9e5716dd67069ed3053ca3)

File: operators/src/util/raster_stream_to_geotiff.rs (hunk 1 above)
Comment (jdroenner) [reply operators/src/util/raster_stream_to_geotiff.rs]: [55358bb](https://github.com/geo-engine/geoengine/pull/473/commits/55358bb33360ff88ff9e5716dd67069ed3053ca3)

**Review by ChristianBeilschmidt — APPROVED** [review]

**Comment by ChristianBeilschmidt** [comment]
bors merge

**Pull request merged**